jobserver/templates/workspace_detail.html

```diff
@@ -90,7 +90,16 @@ <h3>{{ workspace.name }}</h3>
 
     <pre class="text-left">{{ actions_error }}</pre>
 
-    <p>Fix the error above and reload this page to continue.</p>
+    <p>
+      To run actions for this Workspace please fix the error above and reload
+      this page to continue.
+    </p>
+
+    <p>
+      If the branch no longer exists you will need to set up a new workspace.
+      Previous jobs in this workspace are still available via the Logs button.
```

Comment:
If the workspace is unarchived, should we also suggest archiving it?

Reply:
Yeah, good point, this _was_ the impetus to add that feature…